spec/models/user_spec.rb

```diff
@@ -187,6 +187,11 @@ def user_from_authorization_service(service_name, signed_in_resource, cta_varian
         user.mastodon_url = "mastodon.social/@test"
         expect(user).not_to be_valid
       end
+
+      it "does not raise InvalidURIError when url is badly formatted" do
+        user.mastodon_url = "ben .com"
+        expect { user.validate }.not_to raise_error(URI::InvalidURIError)
```

Comment:
Maybe we can test for the presence of the error instead of just testing that we don't raise an exception, WDYT?

Reply:
It makes sense for me. I will change it. Thanks!